docs/administering/sniproxy.md

```diff
@@ -0,0 +1,146 @@
+
+```bash
+sudo nano /etc/sniproxy.conf
+```
+
+Here is the example configuration for `/etc/sniproxy.conf`:
+
+```
+# sniproxy.conf
+# Setup for sharing port 443 with Sandstorm
+
+user daemon
+pidfile /var/run/sniproxy.pid
+
+error_log {
+    syslog daemon
+    priority notice
+}
+
+listen 443 {
+    proto tls
+    table https_hosts
+    fallback 127.0.0.1:7443
+
+    access_log {
+        filename /var/log/sniproxy/https_access.log
+        priority notice
+    }
+}
+
+table https_hosts {
+    .*\.sandcats\.io 127.0.0.1:6443
+}
+```
+
+### Startup
+
+We'll have to ensure sniproxy starts when the system is rebooted. For that we'll need to ensure it is enabled in ```/etc/default/sniproxy``` (```ENABLED=1```). You may also need to run this command to make sure sniproxy will automatically start on boot up:
+
+```bash
+sudo update-rc.d sniproxy enable
+```
+
+## Setting up Sandstorm
+
+Enabling TLS support on Sandstorm is out of the scope of this tutorial. If you don’t have it set up yet, head to [the HTTPS guide](ssl.md).
+
+We need to change Sandstorm's configuration so that it listens on the port that sniproxy is forwarding the requests to (6443, in this example). We make sure BASE_URL and WILDCARD_HOST use URLs on port 443. Since that is the standard port for HTTPS, we don't need :443 at the end. We also adjust the BIND_IP. We'll assume you have `example.sandcats.io` as your Sandstorm address in the configuration example; you will replace this with your own Sandcats hostname.
+
+```bash
+sudo nano /opt/sandstorm/sandstorm.conf
+```
+
+Relevant contents of `/opt/sandstorm/sandstorm.conf`:
+
+```
+# Bind localhost to avoid anyone connecting directly to Sandstorm
+BIND_IP=127.0.0.1
+# No ports here, standard HTTPS (port 443)
+BASE_URL=https://example.sandcats.io
+WILDCARD_HOST=*.example.sandcats.io
+# This is the port sniproxy will connect to
+HTTPS_PORT=6443
+```
+
+## Setting up your current web server
+
+This will depend on the server you have, typically Apache or nginx. Remember the accessible URLs will still use the standard HTTPS port; this change is only made to allow the sniproxy to sit in the middle.
+
+If you are using nginx, you can change all the configuration files using ```sed``` to replace 443 with 7443. To do that, run this command:
+
+```bash
+sudo sed -ri 's/443/7443/g' /etc/nginx/sites-available/*
+```
+
+Keep in mind this might not work for you if you're using '443' anywhere in the configuration files that is not to refer the port.
+
+## Final steps: put it to work
+
+Now is the time to see if it worked. Shutdown your web server and Sandstorm and start them again. Start sniproxy as well.
+
+For example, if you are using nginx, you can run:
+
+```bash
+sudo service nginx stop
+sudo service sandstorm stop
+sudo service sniproxy start
+sudo service sandstorm start
+sudo service nginx start
+```
+
+You probably have to fix the Sandstorm administrator OAuth token. Run the following and follow the instructions that it prints out:
```

Comment:
I'm not sure I follow why this would be necessary. Also, the admin token has nothing to do with OAuth afaik.

Reply:
I agree we can remove this, as we are removing the assumption the BASE_URL is being changed in this tutorial. (The reason for admin-token is presumably that if someone invalidated their Google/GitHub login, they need to run admin-token to get into the admin panel to correct their OAuth configuration.)